### PR TITLE
chore: make `env upgrade` a no op

### PR DESCRIPTION
**Pull request by efekarakus**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/cli/env_deploy.go [hunk 1]
```diff
@@ -204,7 +204,6 @@ func buildEnvDeployCmd() *cobra.Command {
 		Example: `
 Deploy an environment named "test".
 /code $copilot env deploy --name test`,
-		Hidden: true,
```
Comment (dannyrandall) [inline internal/pkg/cli/env_deploy.go:207]: :tada:

File: internal/pkg/cli/env_upgrade.go [hunk 2]
```diff
@@ -4,530 +4,28 @@
...
-	vars := envUpgradeVars{}
+	var envName, appName string
+	var all bool
 	cmd := &cobra.Command{
-		Use:    "upgrade",
-		Short:  "Upgrades the template of an environment to the latest version.",
+		Use:   "upgrade",
+		Short: "Upgrades the template of an environment to the latest version.",
+		Long: `Deprecated. Use "copilot env deploy" instead.
+This command is now a no op. Used to upgrade the template of an environment stack to the latest version.
```
Comment (dannyrandall) [inline internal/pkg/cli/env_upgrade.go]: both definitions of `Used` work here lol

File: internal/pkg/cli/env_upgrade.go [hunk 3]
```diff
@@ -4,530 +4,28 @@
...
 // the environment template.
 func buildEnvUpgradeCmd() *cobra.Command {
-	vars := envUpgradeVars{}
+	var envName, appName string
+	var all bool
 	cmd := &cobra.Command{
-		Use:    "upgrade",
-		Short:  "Upgrades the template of an environment to the latest version.",
+		Use:   "upgrade",
+		Short: "Upgrades the template of an environment to the latest version.",
```
Comment (dannyrandall) [inline internal/pkg/cli/env_upgrade.go]: should mention deprecated here as well? i don't think the short description will ever print anyways but to be safe

**Review by dannyrandall — APPROVED** [review]

**Review by bvtujo — APPROVED** [review]
Blessed red code 🧧

**Pull request merged**